app/Child.php

```diff
@@ -137,4 +149,33 @@ public function family()
     {
         return $this->belongsTo('App\Family');
     }
+
+    /**
+     * Get info about whether child can defer
+     *
+     * @return boolean
+     */
+    public function getCanDeferAttribute()
+    {
+      $notices = $this->family->registrations[0]->getEvaluator()->evaluations["App\Child"]["notices"];
+      if (!array_key_exists('ScottishChildCanDefer', $notices)) {
+        return false;
+      }
+      $rule = [
```

Comment:
Here we're saying
IF the rules you would evauate this Child don't include the ScottishChildCanDefer,
THEN return false; this child cannot be tested for deferring. 

If they _do_ have ScottishChildCanDefer, then make a new validator that _only_ validates the child against the ScottishChildCanDefer rule.

is this redundant?